### PR TITLE
[wni] Fix unit test changes

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Commit that I missed pushing for #188

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
LGTM

File: tools/windows-node-installer/pkg/cloudprovider/aws/ec2_test.go [hunk 1]
```diff
@@ -225,7 +225,8 @@ func TestAwsProvider_waitUntilPasswordDataIsAvailable(t *testing.T) {
 	for _, tt := range tests {
 		got, err := awsProvider.waitUntilPasswordDataIsAvailable(tt.instanceID)
 		if (err != nil) != tt.wantErr {
-			assert.NoError(t, err, tt.wantErr)
+			assert.Error(t, err, tt.wantErr)
```
Comment (sebsoto) [inline tools/windows-node-installer/pkg/cloudprovider/aws/ec2_test.go:228]: Why are you printing tt.wantErr as a message?

**Comment by mansikulkarni96** [comment]
LGTM

**Pull request merged**